pkg/operator/ceph/config/network.go

```diff
@@ -34,8 +34,10 @@ const (
 	PublicNetworkSelectorKeyName = "public"
 	// ClusterNetworkSelectorKeyName is the network selector key for the ceph cluster network
 	ClusterNetworkSelectorKeyName = "cluster"
-	// Whereabouts Ipam type
-	WhereaboutsIpamType = "whereabouts"
+	// WhereaboutsIPAMType is Whereabouts IPAM type
+	WhereaboutsIPAMType = "whereabouts"
+	hostLocalIPAMType   = "host-local"
+	staticIPAMType      = "static"
```

Comment:
Why is `whereabouts` documented and exported but `host-local` and `static` are not?

Reply:
Yes, we don't need them to be exported